test/integration/kubernetes_deploy_test.rb

```diff
@@ -506,7 +506,7 @@ def test_failed_deploy_to_nonexistent_namespace
     original_ns = @namespace
     @namespace = 'this-certainly-should-not-exist'
     assert_deploy_failure(deploy_fixtures("hello-cloud", subset: ['configmap-data.yml']))
-    assert_logs_match(/Result: FAILURE.*Namespace this-certainly-should-not-exist not found/m)
+    assert_logs_match(/namespaces "this-certainly-should-not-exist" not found/m)
```

Comment:
This changed in `kubectl`?

Reply:
This changed in how I am surfacing the error. Since we surface the actual `err` instead of just saying namespace not found, it changes how the logs look

Reply:
Please add back the `Result: FAILURE.*` bit... it proves we collected this info into the result section specifically.